java-grpc/flatbuffers-compiler/src/assembly/windows.xml

```diff
@@ -0,0 +1,17 @@
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+<id>distribution-windows</id>
+  <includeBaseDirectory>false</includeBaseDirectory>
+  <formats>
+    <format>zip</format>
+  </formats>
+  <fileSets>
+    <fileSet>
+      <directory>${basedir}/../..</directory>
+      <includes>
+        <include>flatc.exe</include>
```

Comment:
If this means it is going to require me to build these on 3 platforms, then pull them back to Linux to be able to run Maven, then that is very inconvenient, and not sure that is something I want to commit to.